lib/inflecto/defaults.rb

```diff
@@ -54,5 +54,5 @@
   inflect.irregular('move', 'moves')
   inflect.irregular('cow', 'cows')
 
-  inflect.uncountable(%w(hovercraft moose milk rain Swiss grass equipment information rice money species series fish sheep jeans))
+  inflect.uncountable(%w(hovercraft moose milk rain swiss grass equipment information rice money species series fish sheep jeans))
```

Comment:
I think the lowercase word `swiss` does not exist in english, where the capitalized `Swiss` exists as a name that is always capitalized. We should keep capitalization in this case.